src/google/cloud/ndb/_datastore_api.py

```diff
@@ -469,27 +474,26 @@ def _get_commit_batch(transaction, options):
     Args:
         transaction (bytes): The transaction id. Different transactions will
             have different batchs.
-        options (Dict[str, Any]): Options for the batch. Only "transaction" is
-            supported at this time.
+        options (_options.Options): Options for the batch. Only "transaction"
+            is supported at this time.
 
     Returns:
         _TransactionalCommitBatch: The batch.
     """
     # Support for different options will be tricky if we're in a transaction,
     # since we can only do one commit, so any options that affect that gRPC
     # call would all need to be identical. For now, only "transaction" is
-    # suppoorted if there is a transaction.
-    options = options.copy()
-    options.pop("transaction", None)
-    for key in options:
-        raise NotImplementedError("Passed bad option: {!r}".format(key))
+    # supported if there is a transaction.
+    for key, value in options.items():
```

Comment:
It looks like this takes out the .pop("transaction") part; is that functionality implemented elsewhere?

Reply:
"transaction" is not a member of Options, so it wouldn't show up in this spot. Carlos is correct that the comments were lagging the code a little bit as a result of some refactoring.

See: d6312c5 for corrected comments.